src/HW_models/NRF_EGU.h

```diff
@@ -8,12 +8,24 @@
 #define _NRF_HW_MODEL_EGU_H
 
 #include "nrfx.h"
+#include "hal/nrf_egu.h"
 
 #ifdef __cplusplus
 extern "C"{
 #endif
 
-extern NRF_EGU_Type NRF_EGU_regs[6];
+extern NRF_EGU_Type NRF_EGU_regs[EGU_COUNT];
+
+#define reg_to_id(p_reg)    ((size_t)p_reg - (size_t)(&NRF_EGU_regs[0])) / sizeof(NRF_EGU_Type)
```

Comment:
1. The coding style convention is for macros to be in capital letters
2. This macros are placed in a public header. But, do you need them here?

Reply:
Transformed to uppercase, and moved to NRF_EGU.c file